packages/design-system-tokens/src/tokens/System.Value.json

```diff
@@ -2941,6 +2941,27 @@
       }
     },
     "family": {
+      "lexend": {
+        "$value": [
+          "Lexend",
+          "Open Sans",
```

Comment:
just noticed this... this should be removed as you wouldn't have another web font as a fallback font - should be lexend + system supported fonts for fallbacks in case lexend doesn't load.